### ISSUE TITLE
External Local Storage not opening without intervention on 13.0.0.1beta

### ISSUE THREAD

**Issue by tfmm** [nextcloud/server#7388]
### Steps to reproduce
1. Open files App
2. Attempt to open a directory that is setup as an external local storage

### Expected behaviour
Directory should open

### Actual behaviour
Directory does not open unless I first select the "Sharing" tab in the details panel

## Server configuration detail

**Operating system:** Linux 3.10.0-693.5.2.el7.x86_64 #1 SMP Fri Oct 20 20:32:50 UTC 2017 x86_64

**Webserver:** nginx/1.12.2 (fpm-fcgi)

**Database:** mysql 10.1.29

**PHP version:** 7.0.26
Modules loaded: Core, date, libxml, openssl, pcre, zlib, filter, hash, Reflection, SPL, session, standard, cgi-fcgi, bz2, calendar, ctype, curl, dom, mbstring, fileinfo, ftp, gd, gettext, iconv, imap, json, exif, mcrypt, mysqlnd, PDO, apcu, posix, shmop, SimpleXML, sockets, sqlite3, sysvmsg, sysvsem, sysvshm, tokenizer, xml, xmlwriter, xsl, mysqli, pdo_mysql, pdo_sqlite, wddx, xmlreader, apc, igbinary, memcache, msgpack, xattr, zip, Phar, memcached, mysql, redis, ionCube Loader, Zend OPcache

**Nextcloud version:** 13.0.0 Beta 1 - 13.0.0.6

**Updated from an older Nextcloud/ownCloud or fresh install:** 

**Where did you install Nextcloud from:** unknown

<details><summary>Signing status</summary>

Array
</details>

<details><summary>List of activated apps</summary>

```
Enabled:
 - activity: 2.6.1
 - admin_audit: 1.3.0
 - bookmarks: 0.10.1
 - calendar: 1.5.6
 - comments: 1.3.0
 - contacts: 2.0.1
 - dav: 1.4.5
 - federatedfilesharing: 1.3.1
 - federation: 1.3.0
 - files: 1.8.0
 - files_automatedtagging: 1.3.0
 - files_external: 1.4.1
 - files_markdown: 2.0.1
 - files_pdfviewer: 1.2.0
 - files_sharing: 1.5.0
 - files_texteditor: 2.5.1
 - files_trashbin: 1.3.0
 - files_versions: 1.6.0
 - files_videoplayer: 1.2.0
 - firstrunwizard: 2.2.1
 - gallery: 18.0.0
 - groupfolders: 1.1.0
 - issuetemplate: 0.3.0
 - logreader: 2.0.0
 - lookup_server_connector: 1.1.0
 - metadata: 0.6.0
 - nextcloud_announcements: 1.2.0
 - notes: 2.3.2
 - notifications: 2.1.0
 - oauth2: 1.1.0
 - password_policy: 1.3.0
 - provisioning_api: 1.3.0
 - qownnotesapi: 17.5.0
 - richdocuments: 1.12.39
 - serverinfo: 1.3.0
 - sharebymail: 1.3.0
 - survey_client: 1.1.0
 - systemtags: 1.3.0
 - theming: 1.4.1
 - twofactor_backupcodes: 1.2.3
 - twofactor_u2f: 1.4.0
 - updatenotification: 1.3.0
 - workflowengine: 1.3.0
Disabled:
 - bruteforcesettings
 - deck
 - encryption
 - flowupload
 - impersonate
 - keeweb
 - nextant
 - socialsharing_email
 - tasks
 - user_external
 - user_ldap

```
</details>

<details><summary>Configuration (config/config.php)</summary>

```
{
    "instanceid": "***REMOVED SENSITIVE VALUE***",
    "passwordsalt": "***REMOVED SENSITIVE VALUE***",
    "secret": "***REMOVED SENSITIVE VALUE***",
    "trusted_domains": [
        "***REMOVED SENSITIVE VALUE***"
    ],
    "datadirectory": "***REMOVED SENSITIVE VALUE***",
    "overwrite.cli.url": "***REMOVED SENSITIVE VALUE***",
    "dbtype": "mysql",
    "version": "13.0.0.6",
    "dbname": "***REMOVED SENSITIVE VALUE***",
    "dbhost": "***REMOVED SENSITIVE VALUE***",
    "dbtableprefix": "oc_",
    "dbuser": "***REMOVED SENSITIVE VALUE***",
    "dbpassword": "***REMOVED SENSITIVE VALUE***",
    "logtimezone": "UTC",
    "installed": true,
    "loglevel": 1,
    "maintenance": false,
    "appstore.experimental.enabled": true,
    "mail_smtpmode": "smtp",
    "mail_from_address": "***REMOVED SENSITIVE VALUE***",
    "mail_domain": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpauthtype": "LOGIN",
    "mail_smtpauth": 1,
    "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpport": "587",
    "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
    "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
    "mail_smtpsecure": "tls",
    "theme": "",
    "updater.release.channel": "beta",
    "memcache.local": "\\OC\\Memcache\\APCu",
    "memcache.distributed": "\\OC\\Memcache\\Memcached",
    "memcached_servers": [
        [
            "localhost",
            11211
        ]
    ],
    "memcache.locking": "\\OC\\Memcache\\Redis",
    "updater.secret": "***REMOVED SENSITIVE VALUE***"
}
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

## Client configuration

**Browser:** Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3202.94 Safari/537.36

**Operating system:** 

## Logs

<details><summary>Nextcloud log</summary>

```
V9u63eeVliM%2FM6EB5U%3D%3Ao%2FpDHsO4ogYnBmEbpFTt5BaCehVBkhOxHQ7416%2FrT7o%3D","message":"starting upgrade from 12.0.4.1 to 12.0.4.2","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"12.0.4.1"}
{"reqId":"rgZQYl4vbcY2ih6lWYOl","level":0,"time":"2017-12-04T13:10:31+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"--","app":"core","method":"GET","url":"\/core\/ajax\/update.php?requesttoken=umOH7%2B%2BnwQvlqRCdfCEPQrQDRlGTIYKywC6RVjd6GA8%3D%3A7hTm197KsX2J8X%2F%2FBVZMB%2BBRDSTAFPDolwHUYAAPT1k%3D","message":"starting upgrade from 12.0.4.2 to 13.0.0.6","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"12.0.4.2"}
{"reqId":"HwErJgq7t4x7eGhNmQBe","level":3,"time":"2017-12-04T13:13:43+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"core","method":"POST","url":"\/index.php\/settings\/ajax\/enableapp.php","message":"App \"Deck\" cannot be installed because it is not compatible with this version of the server.","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"gjFCzKx4tYSQCE4ucCkC","level":3,"time":"2017-12-04T13:14:00+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"core","method":"POST","url":"\/index.php\/settings\/ajax\/enableapp.php","message":"App \"Deck\" cannot be installed because it is not compatible with this version of the server.","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"AOdvhEMsaNedRKLq18Jq","level":3,"time":"2017-12-04T13:18:13+00:00","remoteAddr":"","user":"--","app":"PHP","method":"--","url":"--","message":"Undefined offset: 1 at \/srv\/www\/drive.REDACTED.co\/public_html\/apps\/admin_audit\/lib\/Actions\/Console.php#32","userAgent":"--","version":"13.0.0.6"}
{"reqId":"DkNOA1UD5sZqqFmyXzYb","level":3,"time":"2017-12-04T13:42:36+00:00","remoteAddr":"","user":"--","app":"core","method":"--","url":"--","message":"Following symlinks is not allowed ('\/data\/ocdata\/web\/REDACTED\/files_trashbin\/files\/kChat.app.d1506534969\/Contents\/Resources\/wineprefix\/dosdevices\/z:' -> '\/\/' not inside '\/data\/ocdata\/web\/REDACTED\/')","userAgent":"--","version":"13.0.0.6"}
{"reqId":"FOZoWZKm79zABfr7kqEd","level":3,"time":"2017-12-04T16:32:12+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"PHP","method":"GET","url":"\/settings\/admin","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"IDouCG6YfDM2Du8Z0jYN","level":3,"time":"2017-12-04T16:32:12+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"--","app":"PHP","method":"GET","url":"\/cron.php","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"mzqxPbx1xqt1EwBo3w2k","level":3,"time":"2017-12-04T16:32:12+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"PHP","method":"PROPFIND","url":"\/remote.php\/webdav","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"jXx2Qo3r4lWyyrlKZAb9","level":3,"time":"2017-12-04T16:32:13+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"PHP","method":"GET","url":"\/heartbeat","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"W8xlbSuWctcKFFjsdUBB","level":3,"time":"2017-12-04T16:32:16+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"PHP","method":"GET","url":"\/settings\/ajax\/checksetup","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"t7GdQ8T9uUlOIEIfYPiJ","level":3,"time":"2017-12-04T16:32:35+00:00","remoteAddr":"69.167.144.138","user":"mobile","app":"PHP","method":"PROPFIND","url":"\/remote.php\/dav\/files\/mobile\/","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (Linux) mirall\/2.3.3 (Nextcloud)","version":"13.0.0.6"}
{"reqId":"3i32q8sezlfTcH3J5mh4","level":3,"time":"2017-12-04T16:33:05+00:00","remoteAddr":"69.167.144.138","user":"mobile","app":"PHP","method":"PROPFIND","url":"\/remote.php\/dav\/files\/mobile\/","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (Linux) mirall\/2.3.3 (Nextcloud)","version":"13.0.0.6"}
{"reqId":"6ojsFyJgXPStKjLWISli","level":3,"time":"2017-12-04T16:33:08+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"work","app":"PHP","method":"PROPFIND","url":"\/remote.php\/dav\/files\/work\/","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (Linux) mirall\/2.3.3 (Nextcloud)","version":"13.0.0.6"}
{"reqId":"X7ayiij0VIcYdMYN2g5R","level":3,"time":"2017-12-04T16:33:38+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"work","app":"PHP","method":"PROPFIND","url":"\/remote.php\/dav\/files\/work\/","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (Linux) mirall\/2.3.3 (Nextcloud)","version":"13.0.0.6"}
{"reqId":"LQMg4ZZdYF6eqyRtTQlF","level":3,"time":"2017-12-04T16:34:17+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"core","method":"POST","url":"\/index.php\/settings\/ajax\/enableapp.php","message":"App \"Deck\" cannot be installed because it is not compatible with this version of the server.","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"Zm07l0UoxCeVJw1XTrZz","level":3,"time":"2017-12-04T16:34:43+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"PHP","method":"GET","url":"\/apps\/twofactor_u2f\/settings\/state","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"0Fcwp4L6ZfurNoAyvq31","level":3,"time":"2017-12-04T16:41:33+00:00","remoteAddr":"69.167.144.138","user":"REDACTED","app":"PHP","method":"GET","url":"\/js\/core\/merged-share-backend.js?v=b04b59fe83519526a5aebbb8fff0a170-6","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"aIAtgQiwU02mP05pbfi6","level":3,"time":"2017-12-04T16:41:33+00:00","remoteAddr":"69.167.144.138","user":"REDACTED","app":"PHP","method":"GET","url":"\/css\/core\/8f175a0240e58a42cfb009638dbf79b6-server.css?v=b04b59fe83519526a5aebbb8fff0a170-6","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"j8atCiVhc53GyXXCzvhf","level":3,"time":"2017-12-04T16:41:34+00:00","remoteAddr":"69.167.144.138","user":"REDACTED","app":"PHP","method":"GET","url":"\/js\/systemtags\/merged.js?v=b04b59fe83519526a5aebbb8fff0a170-6","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"4thNJXZhqdnJMQgxhSXr","level":3,"time":"2017-12-04T16:41:35+00:00","remoteAddr":"69.167.144.138","user":"REDACTED","app":"PHP","method":"GET","url":"\/js\/core\/merged.js?v=b04b59fe83519526a5aebbb8fff0a170-6","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"BkIfih5P4jIktIOsK9x5","level":3,"time":"2017-12-04T17:32:34+00:00","remoteAddr":"","user":"--","app":"PHP","method":"--","url":"--","message":"Undefined offset: 1 at \/srv\/www\/drive.REDACTED.co\/public_html\/apps\/admin_audit\/lib\/Actions\/Console.php#32","userAgent":"--","version":"13.0.0.6"}
{"reqId":"6Y5O5UM2wieaUd4KESIt","level":3,"time":"2017-12-04T17:34:24+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"no app in context","method":"POST","url":"\/index.php\/settings\/ajax\/enableapp.php","message":"issuetemplate","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"uuBYy0t1sw7c9DPLhfyA","level":3,"time":"2017-12-04T17:34:29+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"no app in context","method":"GET","url":"\/settings\/user","message":"issuetemplate","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"JmWEWvwGx4zlsUd15TCJ","level":3,"time":"2017-12-04T17:34:36+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"no app in context","method":"GET","url":"\/settings\/admin\/issuetemplate","message":"issuetemplate","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"JmWEWvwGx4zlsUd15TCJ","level":3,"time":"2017-12-04T17:34:36+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"PHP","method":"GET","url":"\/settings\/admin\/issuetemplate","message":"Array to string conversion at \/srv\/www\/drive.REDACTED.co\/public_html\/apps\/issuetemplate\/lib\/DetailManager.php#104","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"2oRq8X66SVJZoendnFDZ","level":3,"time":"2017-12-04T17:36:17+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"PHP","method":"GET","url":"\/apps\/gallery\/config?extramediatypes=1","message":"Zend OPcache can't be temporary enabled (it may be only disabled till the end of request) at Unknown#0","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"RNFa7S1bBrPlZlXrzA8Z","level":3,"time":"2017-12-04T17:41:58+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"no app in context","method":"GET","url":"\/settings\/user","message":"issuetemplate","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
{"reqId":"90Wl7lNHbBlIswCHAxYQ","level":3,"time":"2017-12-04T17:42:01+00:00","remoteAddr":"2607:fad0:3:4:be74:10bf:77cf:fc24","user":"REDACTED","app":"no app in context","method":"GET","url":"\/settings\/admin\/logging","message":"issuetemplate","userAgent":"Mozilla\/5.0 (X11; Linux x86_64) AppleWebKit\/537.36 (KHTML, like Gecko) Chrome\/62.0.3202.94 Safari\/537.36","version":"13.0.0.6"}
```
</details>

**Comment by KaraokeStu** [comment]
Just to confirm I have the same issue running NextCloud on Apache.
In the browser you cannot click the initial directory, however if you right-click and open in a new tab, all works as expected.

**Comment by blizzz** [comment]
I don't  see anything related in the log, actually.  @rullzer you've got an idea?

**Comment by tfmm** [comment]
After upgrading to Beta 3, I noticed that when the page loads, the "share" and "..." buttons load briefly for the external storage, but then disappear.  This may have been happening prior to the update as well, I'm not sure.  

The buttons, and functionality to open the storage in the same view, look to disappear after this request:


core.js?v=8c2f33cf-6:4 XHR finished loading: GET "https://REDACTED/ocs/v1.php/apps/files_external/api/v1/mounts?format=json".

**Comment by ziqingshi** [comment]
I have the same question,I fix this 3 days,hava no ideas,I'm from China........Very very very want to know how to fix this question.

**Comment by ziqingshi** [comment]
@blizzz  I have the same question,I fix this 3 days,hava no ideas,I'm from China........Very very very want to know how to fix this question.

**Comment by rullzer** [comment]
I have no idea why. But I see similar behavior when adding an SFTP external share.

@icewind1991 any idea?

@danxuliu as js debugger ;)

**Comment by ziqingshi** [comment]
@rullzer  In the firefox,see the source code,add "<a class="action action-menu permanent" href="#" data-action="menu" data-original-title="" title="">" after the External Local Storage "<a></a>",then you can opening  External Local Storage.

**Comment by danxuliu** [comment]
@rullzer If I am not mistaken this is caused by a change in the layout of the main file list. Since that layout change the element selectors used in the _files_external_ app are no longer valid, and thus the events are not correctly bind. I will take care of it ;-)

**Comment by rullzer** [comment]
@danxuliu thanks for looking into this :rocket:

**Comment by danxuliu** [comment]
#7388 should fix this; please test :-)

As a side note and for future reference, the fact that selecting the _Sharing_ tab in the details view made possible to open the directory turns out to be caused by a bug that changes the file model in the file list row. It can still be seen in #7388, as selecting the _Sharing_ tab causes the file list row to lose the _External storage_ icon; something that needs to be taken care of ;-)

**Comment by tfmm** [comment]
The fix in #7658 works for me.

**Comment by MorrisJobke** [comment]
Fixed by #7658
